naresh/src/main/java/com/lftechnology/phpjava/ems/entities/Employee.java

```diff
@@ -0,0 +1,147 @@
+package com.lftechnology.phpjava.ems.entities;
+
+import com.lftechnology.phpjava.ems.enums.Role;
+
+/**
+ * Employee
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 08 2016
+ */
+public class Employee extends User {
+    private static final String TABLE = "employee";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TABLE" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1068)